src/containers/Storage/Disks/Disks.tsx

```diff
@@ -51,12 +53,13 @@ export function Disks({vDisks = [], viewContext}: DisksProps) {
             </Flex>
 
             <div className={b('pdisks-wrapper')}>
-                {vDisks?.map((vDisk) => (
+                {vDisks?.map((vDisk, index) => (
                     <PDiskItem
                         key={vDisk?.PDisk?.StringifiedId}
                         vDisk={vDisk}
                         highlightedVDisk={highlightedVDisk}
                         setHighlightedVDisk={setHighlightedVDisk}
+                        withDCMargin={vDisksWithDCMargins.includes(index)}
```

Comment:
There is grouping only for PDisks here. This is done to keep VDisks relative sizes